client/src/components/MapController/MapController.js

```diff
@@ -30,6 +30,13 @@ var MapController = React.createClass({
     var update = {}; update[param] = selection;
     this.setState(update);
   },
+
+  handleSetArea: function(wkt) {
+    // TODO: Do something more here?
+    // Really just here so it's not invisibly transferred from parent props, not directly needed
```

Comment:
Yeah, it makes it a bit more clear than if it magically appeared out of nowhere.